### DISCUSSION TITLE
How to open the stellarium project in the way of qtcreator?

### DISCUSSION THREAD

**Discussion by tsuiwade** [Stellarium/stellarium#2008] [Q&A] [answered]
First of all thank you very much for your contributions!

I want to ask a simple and personal question.

I have read the BUILDING.md file many times.
It has been compiled successfully in the windows environment and linux environment, but I am not clear about the file structure. I have a deeper perception of the project in the qtcreator environment. So I want to ask a quesiton ——

`How to open the stellarium project in the way of qtcreator?`  as follows.

![image](https://user-images.githubusercontent.com/78476272/140023317-c9518141-b70a-4467-985d-5e616c8e0aa8.png)

Thanks!!

**Comment by alex-w** [comment, marked as answer]
See https://github.com/Stellarium/stellarium/blob/master/BUILDING.md#integrated-development-environment-ide

> Inside QtCreator, you open the CMakeLists.txt inside Stellarium's source directory. Default settings create a debug build with all useful plugins. In the Projects tab (button in vertical left bar), you should at least configure Debug and Release builds.

**Reply by tsuiwade** [reply]
Wow so cool!
I thought I had read the operation guide carefully, but I did not expect that txt can be opened with qc! 
Now the project can be compiled normally. 
Thank you!
